packages/composables/src/getters/cartGetters.ts

```diff
@@ -6,84 +6,109 @@ import {
   AgnosticDiscount,
   AgnosticAttribute
 } from '@vue-storefront/core';
-import type { Cart, CartItem } from '@vue-storefront/vendure-api';
+import type { Order, OrderLine } from '@vue-storefront/vendure-api';
+import { createPrice } from '../helpers/_utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-function getItems (cart: Cart): CartItem[] {
-  return [
-    {}
-  ];
+interface ExtendedCartGetters extends CartGetters<Order, OrderLine> {
+  getItemOptions: (item: OrderLine, filterByAttributeName?: string[]) => AgnosticAttribute[]
 }
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-function getItemName(item: CartItem): string {
-  return 'Name';
-}
+const getItems = (cart: Order): OrderLine[] => {
+  if (!cart.lines) return [];
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-function getItemImage(item: CartItem): string {
-  return 'https://s3-eu-west-1.amazonaws.com/commercetools-maximilian/products/081223_1_large.jpg';
-}
+  return cart?.lines;
+};
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-function getItemPrice(item: CartItem): AgnosticPrice {
-  return {
-    regular: 12,
-    special: 10
-  };
-}
+const getItemName = (item: OrderLine): string => {
+  return item?.productVariant?.name || '';
+};
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-function getItemQty(item: CartItem): number {
-  return 1;
-}
+const getItemImage = (item: OrderLine): string => {
+  return item?.featuredAsset?.preview || '';
+};
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-function getItemAttributes(item: CartItem, filterByAttributeName?: Array<string>): Record<string, AgnosticAttribute | string> {
+const getItemPrice = (item: OrderLine): AgnosticPrice => {
+  if (!item) {
+    return {
+      regular: 2,
```

Comment:
why?

Reply:
You are right removed this at it may cause confusion.